code/data_structures/src/tree/binary_tree/binary_tree/traversal/postorder/postorder.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+
+using namespace std;
+// Part of Cosmos by OpenGenus Foundation
+struct node
+{
+    int data;
+    node *left;
+    node *right;
+};
+
+node *newnode(int data)
+{
+     node *temp = new node;
+     temp->data = data;
+     temp->left = nullptr;
+     temp->right = nullptr;
+
+     return temp;
+}
+
+void post_order(node *root)
+{
+    if(root == nullptr)
+        return;
+
+    post_order(root->left);
+    post_order(root->right);
+    cout << root->data <<" ";
+
+}
+
+int main()
+{
+    node *root = newnode(1);
+    root->left = newnode(2);
+    root->right = newnode(3);
+    root->left->left = newnode(4);
+    root->left->right = newnode(5);
+    root->right->left = newnode(6);
+    root->right->right = newnode(7);
+
+    cout << "Post order traversal of tree is ";
+    post_order(root);
+
+    return 0;
+}
```

Comment:
Please move test code to the test folder.

Reply:
Hey, can you please elaborate on this part? I checked out the test folders of some algorithms, and they contain test file implementation using CATCH framework. So,  am I supposed to remove this driver function, and make a test file for this code? Please clarify. Thanks.